ios/KMMIC/Sources/Constants/Constant+Emoji.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Constant+Emoji.swift
+//  KMMIC
+//
+//  Created by MarkG on 08/05/2023.
+//  Copyright © 2023 Nimble. All rights reserved.
+//
+
+import Foundation
+
+extension Constants.Emoji {
+
+    static let heart = "❤️"
```

Comment:
Is there a new convention that requires us to define the value of the enum `Emoji` in a separate file? 🤔

Reply:
@phongvhd93 I'm not sure, I'm following the current way on the previous ones 😅